software/apps/Makefile

```
@@ -25,7 +25,7 @@ else
 	ALL := $(filter-out systolic/%,$(APPS))
 endif
 
-ALL_LLVM := $(filter-out chest_q16, $(ALL))
+ALL_LLVM := $(filter-out chest_q16 cfft_q16radix2 cfft_q16radix4, $(ALL))
```

I think this should be like this to disable the CI for now properly:

```suggestion
ALL_LLVM := $(filter-out chest_q16 cfft_radix2_q16 cfft_radix4_q16, $(ALL))
```